kuma/static/styles/locales/ja.scss

```diff
@@ -46,9 +46,10 @@ Styles for Japanese
 
 /* Bug 973171 */
 /* stylelint-disable declaration-no-important */
+/** Back ground of Bug 973171 is obsolete already, the fix was for Windows XP, lacks Anti-Alias function **/
 * {
     /* !important required for locale specific override */
-    font-style: normal !important;
+    /* font-style: normal !important; */  /*  !important prevents vars outside of syntaxbox to be italic  */
```

Comment:
Instead of commenting out the code, perhaps it is better to simply remove it from the codebase altogether?

Reply:
It is OK for me, revision comment in github shows the reason of the change for the later references.

Reply:
Ok, if you can make the change and push an update to this PR, I will test it out locally. Thanks @hmatrjp

Reply:
Oh!, did I replied on wrong place!? It is shown as a " ... reviewed ..."
 I modified the change to ja.sccs following the advice from @schalkneethling . Thanks.